controllers/replication.storage/volumereplication_controller.go

```diff
@@ -363,13 +372,34 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	}
 
 	instance.Status.LastCompletionTime = getCurrentTime()
+
+	var requeueForInfo bool
+	if instance.Spec.ReplicationState == replicationv1alpha1.Primary {
+		lastSyncTime, err := r.getVolumeReplicationInfo(vr)
```

Comment:
Just a heads up!

The last sync time can be "0" or empty, i.e it was never synced even once, we need to determine how that value would be reflected. One option is to set it to the lower bound of UTC epoch (I am a bit confused on what [Timestamp](https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/timestamp) holds, UTC epoch (since 1970), or since `0001-01-01T00:00:00Z`)